### PR TITLE
(#1213) Display Utility Page

### LINKED ISSUE #1213: Layout and Positioning subpage: Display
Update the Display page within the Layout and Positioning section of the documentation site with the content needed for NCIDS for All pilot launch.

### Tasks
- [x] Create content in the shared [Word Document](https://nih.sharepoint.com/:w:/r/sites/NCI-OCPLNewCgovUXDesignSystem/_layouts/15/Doc.aspx?sourcedoc=%7B5102C32D-9BAB-4111-B5BB-EF710E800BB0%7D&file=Display.docx&wdLOR=cD6324434-C16A-9149-ACE2-11574CEB287B&action=default&mobileredirect=true)
    - [ ] IA responsibility:
    - [x] Dev responsibility:
    - [ ] Design responsibility:
- [ ] Review content in shared document
- [ ] Create images in shared folder: `<insert link to shared folder>`
- [ ] Convert content and images to markdown
- [x] Add to documentation site 

### Resources
[Doc Site Content Inventory](https://nih.sharepoint.com/:x:/r/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/General/Details%20-%20NCIDS%20Pilot/Documentation%20Site/Doc%20Site%20Content%20Inventory.xlsx?d=wff83f29cd1554e99892fd963404c69a1&csf=1&web=1&e=QZgQLq)

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1213

## Pull Request Details

_Add description_

Closes #<TICKET-NUMBER>

## Author PR Checklist
Items that the author of the PR is responsible for checking before submitted the PR.

### General:
- [x] I have reviewed the acceptance criteria defined in the ticket and ensured the work has been completed.
- [x] The commit message passes [all quality commit message standards](https://github.com/NCIOCPL/ncids/wiki/Creating-Quality-Commit-Messages).
- [x] Unit tests have been updated or created to reflect any javascript changes.
- [x] Storybook scenarios have been updated or created to reflect any html/css/js changes.

### Accessibility:
- [x] WCAG 2.1 Level AA requirements have been met.

### Development:
- [x] Any new or updated javascript code has 100% unit test coverage.
- [x] New or updated breakpoints have regression images.
- [x] Breaking changes have been thoroughly documented in the PR.

## Product Reviewer PR Checklist
Items the product team is responsible for reviewing.

### General:
- [x] There are no unexpected or unapproved regression image changes. 
- [x] Functionality of interactive elements meet the acceptance criteria.
- [x] The product is visually and functionally the same across the different browsers.

### Accessibility:
- [x] AxeDev Tools: there are no new or outstanding accessibility issues introduced in this PR.
- [x] Lighthouse: scores have not noticeably decreased during this PR.
- [ ] Wave: there are no new errors or contrast errors introduced in this PR.

## Design Reviewer PR Checklist
Items the design team is responsible for reviewing.  

### General:
- [x] New or updated features introduced in this PR are developed mobile-first.
- [x] Breakpoint changes and regression images match those breakpoints.
- [x] This PR has been tested in all supported browsers at all breakpoints.

## Developer Reviewer PR Checklist
Items the development team is responsible for reviewing.

### General:
- [ ] New code passes code quality standards set by industry standards.
- [ ] The expected Storybook stories have been added or updated for the new or updated feature.
- [ ] The expected unit tests have been added or updated for the new or updated feature.

### Accessibility:
- [ ] VoiceOver: Described content matches with what was expected.
- [ ] Keyboard navigation: new or updated features and content are navigable via the keyboard.

### CODE REVIEW

**Comment by sarinapadilla** [comment]
@karlikpj, when I scroll through the sticky positioning example, only the first heading is getting the styling. Headings 2, 3, and 4 are losing the styling.

**Review by sarinapadilla — APPROVED** [review]
Sticky headings are fixed. All of the utilities from the [spreadsheet](https://nih.sharepoint.com/:x:/r/sites/NCI-OCPLNewCgovUXDesignSystem/_layouts/15/Doc.aspx?sourcedoc=%7BAEAC153A-696A-4419-941B-142CC9B627F1%7D&file=USWDS_Utilities.xlsx&action=default&mobileredirect=true) are included. This looks good to me!

**Review by olitharp-nci — CHANGES_REQUESTED** [review]
See https://github.com/NCIOCPL/ncids/pull/1794#pullrequestreview-2208135031

File: docs/content/foundations/layout-positioning/display.mdx [hunk 1]
```diff
@@ -10,33 +10,135 @@ utility_modules:
...
+        - class_name: 'display-none'
+          calculated_value: 'none'
+        - class_name: 'display-table-cell'
+          calculated_value: 'table-cell'
+    utility_examples:
+      - heading:
+        description: |
+        preview: false
+        code: |
+          <section>
```
Comment (olitharp-nci) [inline docs/content/foundations/layout-positioning/display.mdx:36]: A section element is only allowed if there is a heading. There is no heading, remove this. It doesn't appear a wrapper element is needed for this anyway.

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Comment by olitharp-nci** [comment]
Stop resolving conversations. Let the PR reviewer do it.

**Review by olitharp-nci — APPROVED** [review]

**Pull request merged**